pkg/tests/observability_addon_test.go

```diff
@@ -40,8 +40,17 @@ var _ = Describe("Observability:", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(enable).To(Equal(true))
 
-			By("Check ObservabilityAddon is created if there's managed OCP clusters on the hub")
+			By("Check addon resource requirement")
+			res, err := utils.GetMCOAddonSpecResources(testOptions)
```

Comment:
Can you move this test out of `[P2][Sev2][Observability] Modifying MCO cr to disable observabilityaddon (addon/g0)` and add a new one with `[Integration]` metadata added?

Reply:
thanks, I will move it out.